Reflection_Engine/Query/MethodList.cs

```diff
@@ -124,9 +124,17 @@ private static void ExtractAllMethods()
                         }
                     }
                 }
-                catch (Exception)
+                catch (Exception e)
                 {
-                    Compute.RecordWarning("Cannot load types from assembly " + asm.GetName().Name);
+
+                    string message = "Cannot load types from assembly " + asm.GetName().Name + ". Exception message: " + e.Message;
+
+                    if (e.InnerException != null)
```

Comment:
It feels safer if we do:
```suggestion
                    if (!string.IsNullOrEmpty(e.InnerException.Message))
```
because the exception message can be empty in some cases, where we would end up having a trailing `InnerException: ` text with nothing after it.